lib/auth/httpfallback.go

```diff
@@ -16,7 +16,48 @@ limitations under the License.
 
 package auth
 
+import (
+	"context"
+
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport/api/client/proto"
+	wanlib "github.com/gravitational/teleport/lib/auth/webauthn"
+)
+
 // httpfallback.go holds endpoints that have been converted to gRPC
 // but still need http fallback logic in the old client.
 
-// empty for now...
+// REMOVE IN 13.0.0
```

Comment:
```suggestion
// DELETE IN 13.0.0
```

For consistency, we usually use `DELETE IN`

Reply:
Good catch 🖖